### ISSUE TITLE
Github page weird text

### ISSUE THREAD

**Issue by eranhirs** [cptactionhank/docker-atlassian-jira-software#27]
I am seeing strange text (looks like keywords) on your github pages, screenshot attached.
Link: https://cptactionhank.github.io/docker-atlassian-jira-software/
![image](https://user-images.githubusercontent.com/3372820/27303156-8166679e-5542-11e7-99c3-c1698cc017f4.png)

**Comment by cptactionhank** [comment]
Yes, it was a very poor choice of using various lorem ipsum text generators for placeholders because writing documentation is hard. I just simply have not had the time to do any redesign, rewrite, nor extend it due to some possible conflict of interest with my employer.

**Comment by borisov-r** [comment]
On the same page when I use the basic configuration and pull the image the ports for docker image and the host machine are swapped. So if you change the port on the host machine it actually changes the port on the docker container and when you try to connect using the browser it gives error 401.
